commands/host.vegman

```diff
@@ -289,6 +289,31 @@ function cmd_config_default {
   config_default_${action} ${yes}
 }
 
+# @sudo cmd_nmi admin
+# @restrict cmd_nmi admin
+# @doc cmd_nmi
+# Emmit NMI interrupt
+#   -y, --yes - Do not ask for confirmation
+function cmd_nmi {
+  local yes=""
+
+  while [[ $# -gt 0 ]]; do
+    case "${1}" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "${1}";;
+    esac
+    shift
+  done
+
+  if [[ -z "${yes}" ]]; then
```

Comment:
It would be nice to print something like:
```
An NMI signal will be now sent to the host.
That may cause host OS crash an reboot if configured so.
```

I would print that before the `if` so that it would be displayed always. If `--yes` is not specified, then the "Are you sure?" question will appear right after that message, which looks like a nice place for it to me.